httputil/httputil.go

```diff
@@ -133,6 +138,31 @@ func parseRetryHeader(value string) (time.Duration, error) {
 	return time.Until(t), nil
 }
 
+// Attempt to open ~/.netrc file and find login/password for given machine (Host).
```

Comment:
Please start the comment with the function name, otherwise our linter will fail.